test-dladdr.c

```diff
@@ -42,5 +94,14 @@ int main()
     result |= check_dladdr( (char*)dladdr-6, "dladdr-6" );
     // offsets
     result |= check_dladdr( (char*)dladdr+6, "dladdr+6" );
+#ifdef _WIN32
```

Comment:
This looks suspicious. Whole dlfcn-win32 project is for windows. So checking for `_WIN32` is useless, it is always truth.

If you have some tests which are not for WIN64 (and only for WIN32), then this check is incorrect as `_WIN32` is defined also on 64bit Windowses.